dockers/docker-fpm-frr/Dockerfile.j2

```diff
@@ -16,6 +16,7 @@ RUN apt-get update   && \
         libc-ares2      \
         iproute2        \
         logrotate       \
+        libyaml-cpp-dev \
```

Comment:
Assuming you don't actually need the development headers at runtime, there shouldn't be any changes needed in this file. Dependency resolution should be able to install the runtime library automatically.

Reply:
@saiarcot895 : not sure if i completely follow this comment. Without this change I was getting run-time error when `fpmsyncd` was running in bgp docker. PR for reference: https://github.com/sonic-net/sonic-swss/pull/3353

Reply:
thanks @saiarcot895 , i have removed the change. Yes, as part of the build of `docker-swss-layer-bookworm.gz` all swss dependency are resolved and we don't need to explicit install on in frr docker.